tests/examples/examples.test.ts

```diff
@@ -51,7 +51,11 @@ const exclude: string[] = [
   !hasEnv("IBM_VLLM_URL") && ["examples/llms/providers/ibm-vllm.ts"],
   !hasEnv("COHERE_API_KEY") && ["examples/llms/providers/langchain.ts"],
   !hasEnv("CODE_INTERPRETER_URL") && ["examples/tools/custom/python.ts"],
-  ["examples/llms/providers/bam.ts", "examples/llms/providers/bam_verbose.ts"],
+  [
+    "examples/llms/providers/bam.ts",
+    "examples/llms/providers/bam_verbose.ts",
+    "examples/llms/providers/bedrock.ts",
```

Comment:
`bedrock.ts` should be ignored only if there is no ENV set.

Reply:
I will add that. Please note that AWS SDK for Javascript can also load auth from the shared credentials file `~/.aws/credentials` (using a default profile).

Reply:
I see. I also think it can be read from ENV. So what about extending our `examples.test.ts` with something like.

```javascript
(hasEnv("XXX") || isFile("~/.aws/credentials")) && 
```

Reply:
Can I leave this to you to decide where to have this function in a shared file utility?

Reply:
Sure.